src/index.js

```diff
@@ -35,12 +36,12 @@ class SVGInline extends Component {
       width,
       classSuffix,
       cleanupExceptions,
-      ...componentProps,
+      ...componentProps
```

Comment:
That style was enforced by the `eslint-config-i-am-meticulous` package. I added an override for objects in the esLintConfig in `package.json`.